RecoLocalCalo/HcalRecProducers/src/HcalSimpleReconstructor.cc

```diff
@@ -239,6 +245,46 @@ void HcalSimpleReconstructor::processUpgrade(edm::Event& e, const edm::EventSetu
     e.put(rec); // put results
   }// End of upgradeHF
 
+  if(HFQIE10_){
+
+    edm::Handle<QIE10DigiCollection> digi;
+    e.getByToken(tok_hfQIE10_, digi);
+
+    // create empty output
+    std::auto_ptr<HFRecHitCollection> rec(new HFRecHitCollection);
+    rec->reserve(digi->size()); 
+
+    // run the algorithm
+    int first = firstSample_;
+    int toadd = samplesToAdd_;
+    for (QIE10DigiCollection::const_iterator i=digi->begin(); i!=digi->end(); i++) {
+      HcalDetId cell = i->id();
+      DetId detcell=(DetId)cell;
+	  
+      //make dataframe
+      QIE10DataFrame frame(*i);
+	  
+      // rof 27.03.09: drop ZS marked and passed digis:
+      if (dropZSmarkedPassed_)
+      if (frame.zsMarkAndPass()) continue;
```

Comment:
please combine in one "if", add an indentation for the second if or put it in "{}" to improve readability.